addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -0,0 +1,587 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from collections import defaultdict
+from tempfile import NamedTemporaryFile
+from urllib3.util.ssl_ import create_urllib3_context, DEFAULT_CIPHERS
+
+from odoo import fields
+
+import math
+import json
+import requests
+import zeep
+
+from odoo import models, _
+
+
+EUSKADI_CIPHERS = DEFAULT_CIPHERS + ":!DH"
```

Comment:
please explain

Reply:
I am adding the comment at the L10nEsHTTPAdapter